.github/workflows/ci.yml

```
@@ -31,4 +35,6 @@ jobs:
         steps:
             - uses: actions/checkout@v4
             - uses: dart-lang/setup-dart@v1.6.4
+              with:
+                sdk: 3.7
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)